tools/mode_defaults.cwl

```diff
@@ -0,0 +1,76 @@
+cwlVersion: v1.0
+class: CommandLineTool
+id: mode_defaults
+doc: "Selects the appropriate defaults based on given the mode"
+requirements:
+  - class: InlineJavascriptRequirement
+baseCommand: echo
+arguments:
+  - position: 0
+    valueFrom: >-
+      Selecting $(inputs.input_mode) default values
+
+inputs:
+  input_mode: {type: {type: enum, name: sex, symbols: ["WGS", "WXS"] }, doc: "Select if this run is WGS or WXS"}
+  exome_flag: {type: 'string?', doc: "Whether to run in exome mode for callers."}
+  cnvkit_wgs_mode: {type: 'string?', doc: "Entering Y will run cnvkit in WGS mode, otherwise it will run in hybrid mode. Defaults to Y in wgs mode."}
+  i_flag: {type: 'string?', doc: "Flag to intersect germline calls on padded regions.  Use N if you want to skip this. Defaults to N in WGS mode."}
+  lancet_padding: {type: 'int?', doc: "Recommend 0 if interval file padded already, half window size if not. Recommended: 0 for WXS; 300 for WGS"}
+  lancet_window: {type: 'int?', doc: "window size for lancet.  default is 600, recommend 500 for WGS, 600 for exome+"}
```

Comment:
So, it might be unclear, after reviewing the test run, I see that window size 500 was used, it should have been 600. The way your test was run, which is our current standard, is actually exome+ mode on WGS data. We don't normally run lancet on WGS directly, just using an exome bed file + hits from strelka2 and mutect2.

Reply:
Do you want the doc changed? That's just a copy/paste from the existing workflows.

Reply:
Right, so the explanation is technically accurate, but I guess I can make it clearer as to why our default clashes with the `WGS` mode...not sure if that is soemthing that should go in the readme/workflow doc, tool doc, or both...

Reply:
Maybe we can make a separate ticket for that, as silly as that may sound

Reply:
Yeah probably something that will accompany the changes when we make the switch to Teja's recommendations.